### PR TITLE
Update Simplified Chinese translation.

### PR DESCRIPTION
**Pull request by njlyf2011**

### CODE REVIEW

File: 7zip.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[Section]
+﻿[Section]
```
Comment (learn-more) [inline 7zip.txt]: What is this change, did you add a BOM there?

**Comment by njlyf2011** [comment]
has been amended.

**Comment by learn-more** [comment]
Thanks!

**Comment by njlyf2011** [comment]
You are welcome .

**Pull request merged**